.eslintrc.json

```diff
@@ -41,6 +41,14 @@
 					"src/**/*.test.*.{ts,js}"
 				]
 			}
+		],
+		"jest/expect-expect": [
```

Comment:
Hm, I'm not able to resolve this, and neither is CI:

> error  Definition for rule 'jest/expect-expect' was not found

Is there a corresponding dev package that needs to be installed?

Reply:
Huh -- I added it per the library documentation but taking it out results in no issues or complaints.

Removing that change entirely.